technical-guide/developer/ui.md

```diff
@@ -660,3 +660,84 @@ You can use Storybook's `<Canvas>` element to showcase specific stories to enric
 When including codeblocks, please add code in Clojure syntax (not JSX).
 
 You can find an example MDX file in the [Buttons docs](https://hourly.penpot.dev/storybook/?path=/docs/buttons-docs--docs).
+
+### Replacing and old component
+
+**Create screenshots for visual tests BEFORE making any changes.**
```

Comment:
Let's use a level 4 heading that is `Write visual regression tests` and rewrite this line in a follow up paragraph as follows:

```
We need to generate the screenshots for the visual regression tests _before_ making any changes, so we can compare the "before substitution" and "after substitution" states.
```